src/lib/coapcore/src/lib.rs

```diff
@@ -81,4 +81,4 @@ mod seccontext;
 pub use seccontext::*;
 
 mod error;
-pub use error::CredentialError;
+pub use credentials::{CredentialError, CredentialErrorDetail as CredentialErrorKind};
```

Comment:
```suggestion
pub use error::{CredentialError, CredentialErrorDetail as CredentialErrorKind};
```

This should still work on the original position -- it will also work through credentials, but it's a re-export there.